backend/app/tests/utils/api_keys.py

```
@@ -0,0 +1,30 @@
+from pathlib import Path
+from typing import Dict
+import pytest
+from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
+
+SEED_DATA_PATH = (
+    Path(__file__).resolve().parents[3] / "app" / "seed_data" / "seed_data.json"
+)
+
+
+def load_seed_data() -> Dict[str, list]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)